src/fann_train.c

```diff
@@ -40,6 +40,10 @@ fann_type fann_activation_derived(unsigned int activation_function, fann_type st
     case FANN_LINEAR_PIECE:
     case FANN_LINEAR_PIECE_SYMMETRIC:
       return (fann_type)fann_linear_derive(steepness, value);
+    case FANN_LINEAR_PIECE_LEAKY:
```

Comment:
Wouldn't be better to call it `FANN_LINEAR_PIECE_RECT_LEAKY`?

Reply:
Yes, I can make that change if you want.

Reply:
Hmm. Let me just change it and we can take it from there.